demo/src/main/java/com/example/demo/repository/EngineerRepository.java

```diff
@@ -0,0 +1,53 @@
+package com.example.demo.repository;
+
+import com.example.demo.bean.Engineer;
+
+import java.util.Arrays;
+
+public class EngineerRepository {
+    public static Engineer[] engineers = new Engineer[3];
+
+    public Engineer[] createEngineer(Engineer engineer) {
+
+        for (int i = 0; i < engineers.length; i++) {
+            if (engineers[i] == null) {
+                engineer.setIdEngineer(count(i));
+                engineers[i] = engineer;
+                if ((i * 2) >= engineers.length) {
+                    Engineer[] engineersTemp = Arrays.copyOf(engineers, (engineers.length + 5));
+                    engineers = engineersTemp;
+                }
+                break;
+            }
+        }
+        return engineers;
+    }
+
+    public Engineer readEngineer(int i) {
```

Comment:
method name "read" , change to "find"

Reply:
Look carefully, I already did this fix
https://github.com/Tutritp01/TgBot/pull/16/commits/13e2e27f764dc13150f3da84d06b5254be035645